plugin/docker/src/main/java/com/perl5/lang/perl/idea/sdk/host/docker/PerlDockerAdapter.java

```diff
@@ -110,6 +110,30 @@ public PerlDockerAdapter(@NotNull PerlDockerData data) {
     return containerName;
   }
 
+  private boolean tryCopyWithTar(@NotNull String containerName, @NotNull String remotePath, @NotNull String localPath)
+    throws ExecutionException {
+    File localPathFile = new File(localPath);
+    File remotePathFile = new File(remotePath);
+    String command = String.join(" ", DOCKER_EXECUTABLE, EXEC, INTERACTIVELY, containerName,
+                                 "tar", "hCcF", remotePathFile.getParent(), "-", remotePathFile.getName(),
+                                 "|", "tar", "Cxf", localPathFile.getParent(), "-");
```

Comment:
This is a separate feature, separate commit please with proper explanation.
Also needs test.

Reply:
Again, why do we need to have interactivity here?

Reply:
Interactively I mention in previous comment. 
For test i dont know where are located tests for docker interaction can you point me where to add it.

Reply:
Dam, this may be tricky. Current integration tests supposed to be run with any interpreter - local, wsl or docker. And this is docker-specific test, so probably should be a separate class with unit test.
I have no example for this yet. You may describe the set up and I figure out the test later.